perception/traffic_light_classifier/launch/traffic_light_classifier.launch.xml

```diff
@@ -13,6 +13,7 @@
   <arg name="classifier_type" default="0" unless="$(var use_gpu)"/>
 
   <arg name="build_only" default="false" description="shutdown node after TensorRT engine file is built"/>
+  <arg name="backlight_threshold" default="0.85" description="overwrite signals if the RoI image intensity is greater than this value"/>
```

Comment:
[Q]
Does the value of 0.85 have specific meaning? In which situations does it correspond?

Reply:
> Does the value of 0.85 have specific meaning?

As far as I experimented, with `0.85`, the node works well to overwrite traffic signal with unknown in harsh backlight situation. 
As the described above, I think `0.85` is higher enough not to disturb the node's behavior in normal backlight.   

> In which situations does it correspond?

If the value becomes much higher, the node only works in the harsher backlight situations.

Reply:
Thanks!
Could you note that to README?

Reply:
I updated README by https://github.com/autowarefoundation/autoware.universe/pull/5852/commits/fa3a584ce8c8513fe877c66dc5b2fd2b3ca2b412